### PR TITLE
add endpoint to get LAN IP address

### PR DESCRIPTION
**Pull request by kdomanski**

### CODE REVIEW

**Comment by tiff** [comment]
:+1: makes sense

File: app/index.js [hunk 1]
```diff
@@ -110,6 +111,12 @@ var main = function () {
       next();
     });
   });
+  app.get('/lanip', function(req, res, next) {
+    var iproute = spawn("ip", ["route", "get", "8.8.8.8"], [null, 'pipe', 'ignore']);
```
Comment (hinnerk) [inline app/index.js]: The network frequently fails so the API should return a 404 with details for cases like this:

``` sh
$ ip route get 8.8.8.8
RTNETLINK answers: Network is unreachable
```

File: app/index.js [hunk 2]
```diff
@@ -110,6 +111,12 @@ var main = function () {
       next();
     });
   });
+  app.get('/lanip', function(req, res, next) {
```
Comment (hinnerk) [inline app/index.js]: Could you make this path `/v1/network/default/ip`?

**Comment by hinnerk** [comment]
Requires experimental-platform/platform-configure/pull/37

**Pull request merged**